docs/source/en/model_doc/albert.mdx

```diff
@@ -23,14 +23,14 @@ speed of BERT:
 
 The abstract from the paper is the following:
 
-*Increasing model size when pretraining natural language representations often results in improved performance on
+_Increasing model size when pretraining natural language representations often results in improved performance on
 downstream tasks. However, at some point further model increases become harder due to GPU/TPU memory limitations,
 longer training times, and unexpected model degradation. To address these problems, we present two parameter-reduction
 techniques to lower memory consumption and increase the training speed of BERT. Comprehensive empirical evidence shows
 that our proposed methods lead to models that scale much better compared to the original BERT. We also use a
 self-supervised loss that focuses on modeling inter-sentence coherence, and show it consistently helps downstream tasks
 with multi-sentence inputs. As a result, our best model establishes new state-of-the-art results on the GLUE, RACE, and
-SQuAD benchmarks while having fewer parameters compared to BERT-large.*
+SQuAD benchmarks while having fewer parameters compared to BERT-large._
```

Comment:
Could you revert these changes? 🙏

Reply:
Does that mean deleting "_"?

Reply:
It means you can leave the asterisks `*` instead of using an underscore `_`